packages/components/src/DateTimePickers/DateRange/Manager/Manager.component.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import { DateRangeContext } from '../Context';
+import {
+	extractRangeParts,
+	extractRangePartsFromTextInput,
+	extractRangePartsFromDate,
+} from '../date-range-extraction';
+
+function ContextualManager(props) {
+	function getOptions() {
```

Comment:
You can replace this function by just a const.
```suggestion
	const options = { dateFormat: props.dateFormat };
```